core/src/test/java/org/elasticsearch/routing/AliasRoutingIT.java

```diff
@@ -77,7 +183,7 @@ public void testAliasCrudRouting() throws Exception {
         logger.info("--> updating with id [1] and routing through alias");
         client().prepareUpdate("alias0", "type1", "1")
                 .setUpsert(XContentFactory.jsonBuilder().startObject().field("field", 1).endObject())
-                .setScript(new Script("ctx._source.field = 'value2'", ScriptService.ScriptType.INLINE, null, null))
+                .setScript(new Script("", ScriptService.ScriptType.INLINE, MockScriptEngine.NAME, Collections.singletonMap("field", "value2")))
```

Comment:
Having worked on this yesterday, I did see that some of the tests will need a mock, but I think we can take out the scripting aspect from this one. The test aims at verifying that whatever update works against an alias. We can then provide a partial document instead of a script like I did in #13945?